src/components/ParksWrapper/ParksWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './ParksWrapper.css';
+
+interface ParksWrapperProps {
+  park: {
+    id: string;
+    fullName: string;
+    description: string;
+    activities: Array<{ id: string; name: string }>;
+  } | null;
+}
+
+function ParksWrapper(props: ParksWrapperProps) {
+  const { park } = props;
+
+  if (!park) {
+    return <div className="error">Park information is not available.</div>;
+  }
```

Comment:
Love that you're thinking about the fact that data might be missing and how to handle it.